app/lib/screens/council_screen.dart

```diff
@@ -14,10 +14,49 @@ class CouncilScreen extends StatefulWidget {
 class _CouncilScreenState extends State<CouncilScreen> {
   final urlController = TextEditingController();
   String? errorMessage;
+  String? selectedNetwork = '';
```

Comment:
This selected network doesn't get updated when the URL is changed manually. It should be updated to change the buttons activity.